### PR TITLE
[nodejs] Update chart image response

### PR DESCRIPTION
**Pull request by chejennifer**
- increase the size of the png image of the chart to improve image quality when consumed by Bard
- add an option to return the chart as an svg to be used by Bard to test consuming our charts as an svg instead of a png

### CODE REVIEW

File: static/src/server.ts [hunk 1]
```diff
@@ -105,6 +104,8 @@ const BARD_ALLOWED_CHARTS = new Set(["LINE", "BAR", "RANKING", "SCATTER"]);
 // The root to use to form the dc link in the tile results
 // TODO: update this to use bard.datacommons.org
 const DC_URL_ROOT = "https://datacommons.org/explore#q=";
+// Size of the PNG to return for the chart query
+const PNG_WIDTH = 1600;
```
Comment (dwnoble) [inline static/src/server.ts:108]: Would it make sense to return SVGs at the same width (maybe set this in the SVG chart itself?) Now SVGs get returned with a 550px width and pngs with 1600

**Review by dwnoble — COMMENTED** [review]
Nice!

The SVG charts look fine in chrome, but in safari & the mac OS preview (when you press "spacebar" when the file is selected in finder) the title gets cut off
Screenshot 2024-01-22 at 10 19 39 AM

File: static/src/server.ts (hunk 1 above)
Comment (chejennifer) [reply static/src/server.ts:108]: The reason for returning pngs at 1600 is to just use a really large size to preserve image quality no matter what size Bard actually chooses to render the image at. I tried drawing the actual svg at 1600 and the scaling makes everything super hard to see. Instead, I updated the returned svg to use viewbox with the original dimensions & caller can then update the width and height to whatever they need.

**Comment by chejennifer** [comment]
> Nice!
> 
> The SVG charts look fine in chrome, but in safari & the mac OS preview (when you press "spacebar" when the file is selected in finder) the title gets cut off Screenshot 2024-01-22 at 10 19 39 AM

thanks for the review! updated the svg charts to work in safari as well 
Screenshot 2024-01-22 at 11 28 36 AM

File: static/src/server.ts (hunk 1 above)
Comment (dwnoble) [reply static/src/server.ts:108]: Sounds reasonable

**Review by dwnoble — APPROVED** [review]

**Pull request merged**